testdata/p4tc_samples_outputs/tc_type_annotation_example_control_blocks.c

```diff
@@ -100,7 +100,7 @@ if (/* hdr->ipv4.isValid() */
                                 {
 /* drop_packet() */
                                     drop_packet();
-                                                                        hdr->ipv4.dstAddr = value->u.MainControlImpl_default_route_drop.ipv4addr;
+                                                                        hdr->ipv4.dstAddr = bpf_htonl(value->u.MainControlImpl_default_route_drop.ipv4addr);
```

Comment:
IPv4 addresses are already in network order in the P4TC side, so there is no need to convert it here

Reply:
addressed